src/selectedtests/evergreen_helper.py

```diff
@@ -15,3 +21,93 @@ def get_evg_project(evg_api: EvergreenApi, project: str) -> Optional[Project]:
         if evergreen_project.identifier == project:
             return evergreen_project
     return None
+
+
+def get_evg_module_for_project(
+    evg_api: EvergreenApi, project: str, module_repo: str
+) -> ManifestModule:
+    """
+    Fetch the module associated with an Evergreen project.
+
+    :param evg_api: An instance of the evg_api client
+    :param project: The name of the evergreen project to analyze
+    :param module_repo: Name of the module to analyze
+    :return: evg_api client instance of the module
+    """
+    version_iterator = evg_api.versions_by_project(project)
+    recent_version = next(version_iterator)
```

Comment:
[nit] You might consider pulling this into a separate function to communicate what is happening here. Maybe something like get_most_recent_version?